src/components/Col.jsx

```diff
@@ -66,31 +66,31 @@ const Col = ({
 		classNames.push(getClass(`col-xl12-${xl12}`));
 	}
 
-	if (xs4Offset) {
+	if (xs4Offset || xs4Offset === 0) {
```

Comment:
I think it might be cleaner to check `xs4Offset != null`, which should cover null or undefined, while permitting a value of zero.  Same with all of the other checks.

Reply:
@kgregory Thanks for the comment! I've updated to your suggestion